### DISCUSSION TITLE
RTL Support

### DISCUSSION THREAD

**Discussion by JWesorick** [nativewind/nativewind#85] [Ideas]
I'm trying to use `rtl:` and `ltr:` modifiers, but they dont seem to work.

https://tailwindcss.com/docs/hover-focus-and-other-states#rtl-support

**Comment by marklawlor** [comment]
I held off on this because its still an experimental feature of Tailwind, but I should be able to slip it into `v2` :+1:

**Comment by marklawlor** [comment]
I'm going to delay this until after the `2.0` release. Tailwind CSS implements the `directionVariants` plugin (which provides RTL) differently to the core plugins which makes it harder to implement.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
